src/main/java/edu/umich/ctools/sectionsUtilityTool/FriendServlet.java

```diff
@@ -0,0 +1,207 @@
+//do ctrl+f to find all "dovek" instances before committing.
+package edu.umich.ctools.sectionsUtilityTool;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Properties;
+import java.util.ResourceBundle;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.json.*;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class FriendServlet extends HttpServlet {
+
+	private static Log M_log = LogFactory.getLog(FriendServlet.class);
+	private static final long serialVersionUID = 7284813350014385613L;
+
+	private static final String FRIEND_CHECK_EXISTS = "some.regex";
+	private static final String FRIEND_CREATE = "some.regex";
+	private static final String FRIEND_SEND_EMAIL = "some.regex";
+	private static final String DELETE = "DELETE";
+	private static final String POST = "POST";
+	private static final String GET = "GET";
+	private static final String PUT = "PUT";
+	ResourceBundle props = ResourceBundle.getBundle("sectiontool");
+
+	public void init() throws ServletException {
+		M_log.debug(" Servlet init(): Called");
+	}
+
+	protected void doGet(HttpServletRequest request,HttpServletResponse response){
```

Comment:
This doesn't look like a correct implementation of REST.  Let's discuss that F2F.